### ISSUE TITLE
Nokia Software updater

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#608]
**Reported by Agent Golder on 2010-03-30 08:44**
NVDA doesn't read the dialogues but only the buttons navigating with tab. While of course, with other screen readers performance is better...

**Comment by dkager** [comment]
It would seem that this has become somewhat obsolete by the moving on of time. :)
